prime-router/src/test/kotlin/metadata/MapperTests.kt

```diff
@@ -80,6 +82,178 @@ class MapperTests {
         assertThat(mapper.apply(lookupElement, args, elementAndValues).value).isEqualTo("y")
     }
 
+    private fun assertEq(
+        mapper: Mapper,
+        elm: Element,
+        args: List<String>,
+        EAVs: List<ElementAndValue>,
+        compVal: String
+    ) {
+        assertThat(
+            mapper.apply(elm, args, EAVs).value
+        ).isEqualTo(compVal)
+    }
+
+    @Test
+    fun `test IfThenElseMapper`() {
```

Comment:
I could no determine the difference in your unit tests from the name of the function.  Maybe add more to the name.

Reply:
`test basic element value functionality IfThenElseMapper`()